
async function main()
{

  console.log("Hello, World!");

  return 0;
}

main().catch(err => console.log(err));